packages/twenty-docker/prod/docker-compose.yml

```diff
@@ -17,8 +17,11 @@ services:
       ENABLE_DB_MIGRATIONS: true
 
       SIGN_IN_PREFILLED: ${SIGN_IN_PREFILLED}
-      STORAGE_TYPE: local
+      STORAGE_TYPE: ${STORAGE_TYPE}
       STORAGE_LOCAL_PATH: .local-storage
```

Comment:
This is the default setting in app. Feels a bit strange to hardcode it a second time at this layer. Either we convert it to an env variable like we did for  STORAGE_TYPE or we remove it?

Reply:
Fixed in https://github.com/twentyhq/twenty/pull/4613/commits/e6fc5cbecd09723c64e3df134b133bc8b84da01a